README.md

```diff
@@ -47,26 +47,103 @@ spec:
 ## How does the intents operator work?
 
 ### Network policies
-The intents operator automatically creates, updates and deletes network policies, and automatically labels client and server pods, 
-to match declarations in client intents files.
-The policies created are `Ingress`-based, so source pods are labeled with a `can-access-<target>=true` 
-while destination pods are labeled with `has-identity=<target>`.
+The intents operator automates the creation, management, and deletion of network policies based on user-defined client intents. To track the services it manages, it assigns labels to pods. These labels are derived from the pod's metadata, specifically combining the name of its top-level owner (e.g., a Kubernetes deployment) and the namespace where it's deployed. This unique identifier is then used within the generated network policies. The operator handles both **Ingress** and **Egress** traffic.
 
-The example above results in the following network policy being created: 
+In the network policy detailed below, we can see an example of Ingress definitions. Two separate policies are defined for the backend and frontend services within the `otterize-example` namespace, each designed to strictly control ingress traffic based on pod labels.
```

Comment:
This is confusing to me as it's talking about Ingress definitions, but the immediate association is with an Ingress resource. Could we say a NetworkPolicy with type Ingress instead?